app/components/participant-creator/component.js

```diff
@@ -73,6 +73,12 @@ export default Ember.Component.extend({
             });
             if (!useAsPassword) {
                 password = makeId(10);
+                this.get('extra').pushObject({
+                    key: 'password',
+                    value: password
+                });
+                extra['password'] = password;
+                this.set('useAsPassword', 'password');
```

Comment:
Does that imply that our accounts have a default password of "password"?

We should not do that. Preliminary experiments show that accounts actually get a randomly generated password, which is much better- but it means that this value is confusing.